DEV_GUIDE.md

```diff
@@ -60,6 +60,14 @@ Build the project like this:
 mvn clean install
 ```
 
+The project provides some Kroxylicious-maintained Filter implementations that are not built by default. To also build 
```

Comment:
Possibly worth clarifying they are not included in distribution by default we do compile them.

Reply:
Cool, I didn't realise this, have moved it down to be under the section describing the `dist` profile.